browser/src/view/components/Content/ContentCollection.scss

```diff
@@ -0,0 +1,36 @@
+@use '../../../styles/common/variables' as *;
+
+table {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 100%;
+  height: 100%;
+  border-collapse: collapse;
+  border: 1px solid $COLOR-TABLE-BORDER;
+  border-radius: 0.5rem;
+  box-shadow: 0px 4px 8px -2px rgba(16, 24, 40, 0.1), 0px 2px 4px -2px rgba(16, 24, 40, 0.06);
+}
+
+th {
+  height: 2.75rem;
+  background-color: $COLOR-TABLE-HEADER-BG;
+  text-align: left;
+  margin: 0;
+  padding-left: 1.5rem;
+}
+
+tr {
+  height: 4.5rem;
+  border-bottom: 1px solid $COLOR-TABLE-BORDER;
```

Comment:
IMO colors, fonts, box-shadow, etc, should all be associated more with brand rather than component.  The variable names should reflect that.  This way, we can keep colors/fonts/borders/etc all within our brand rather than have a ton of different one-offs.